lib/components/molecules/HeroBanner/HeroBanner.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+/**
+ *
+ * HeroBanner
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import styles from './HeroBanner.style';
+import type { Props } from './types';
+
+const HeroBanner = ({
+  className,
+  children,
+  heroImageWidth,
+  childrenContainerWidth,
+  renderHeroImage,
+  renderHeroCarousel,
+}: Props): Node => {
+  return (
+    <div className={classnames(className)}>
+      <section>
+        <header role="banner">
+          {renderHeroImage && (
+            <div className="hero-image" heroImageWidth={heroImageWidth}>
```

Comment:
No need to repeat hero- in each class we are not following BEM in styled components 
change -> hero-image, hero-carousel, hero-children